azure/scope/cluster.go

```diff
@@ -165,11 +166,17 @@ func (s *ClusterScope) PublicIPSpecs() []azure.PublicIPSpec {
 }
 
 // LBSpecs returns the load balancer specs.
-func (s *ClusterScope) LBSpecs() []azure.LBSpec {
-	specs := []azure.LBSpec{
-		{
+func (s *ClusterScope) LBSpecs() []azure.ResourceSpecGetter {
```

Comment:
/lgtm cancel

whoops I just realized right after approving, we also need to add LoadBalancersReadyCondition to the list of owned conditions